app/api/graphql/route.ts

```diff
@@ -111,9 +119,17 @@ const resolvers: { Query: QueryResolvers; Mutation: MutationResolvers } = {
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     getLikes: async (_parent, { type, id }, context) => {
       try {
-        const username = process.env.GITHUB_USERNAME ?? ''
-        const client = createPublicGitHubClient(username)
-        const likesData = await client.getLikes()
+        let likesData
+        
+        if (process.env.NODE_ENV === 'development') {
```

Comment:
I think we should create a smart client which wraps this logic inside, and it's smart enough to know when it should fetching from Github API, when it should load from /data local file.